app/lib/ui/widgets/list_tile.dart

```diff
@@ -30,7 +30,7 @@ class TemplateListTile extends StatelessWidget {
       child: InkWell(
         onTap: () {
           Navigator.of(context).pop();
```

Comment:
- [ ] I thought we were no longer using context?

I could definitely be wrong. Does this work since we give our main the navigator service as its context?

Reply:
I haven't implemented any kind of services/viewmodels here as I already mentioned